.github/workflows/test_security.yml

```diff
@@ -20,7 +22,7 @@ jobs:
   integ-test-with-security-linux:
     strategy:
       matrix:
-        java: [11, 17, 21]
+        java: [21]
```

Comment:
Only remain JDK 21 in security enabled test CI due to security plugin recently update https://github.com/opensearch-project/security/pull/4457

Reply:
what about BWC?

Reply:
In BWC we don't enable security mode, so actually no need to remove older version JDKs